src/invoker.js

```diff
@@ -7,14 +7,14 @@ var curryN = require('./curryN');
  * Turns a named method with a specified arity into a function
  * that can be called directly supplied with arguments and a target object.
  *
- * The returned function is curried and accepts `len + 1` parameters where
+ * The returned function is curried and accepts `arity + 1` parameters where
  * the final parameter is the target object.
  *
  * @func
  * @memberOf R
  * @category Function
  * @sig (Number, String) -> (a... -> c -> b)
```

Comment:
@davidchambers I agree. That is a much better signature.
@CrossEye We'll invoke a method on the last parameter so it has to be an object.